docs/getting-started.md

```diff
@@ -231,31 +231,14 @@ to connect for the first time. You can setup additional connections using the
 
 ## Step 6: Do something awesome
 
-This will be where we tell you how to use the EV3's main functions, and how
-to write programs. We are still learning, so everything here should be
-considered experimental and subject to major changes.
-
-### Using the EV3 hardware drivers
-
-Here are some guides for using each of the major components.
-
-* [Sensors](/docs/sensors)
-* [Motors](/docs/tutorials/tacho-motors/)
-* [Sound](https://github.com/ev3dev/ev3dev/wiki/Using-Sound)
-* [LCD](/docs/tutorials/using-ev3-lcd/)
-* [Buttons](/docs/tutorials/using-ev3-buttons/)
-* [LEDs](https://github.com/ev3dev/ev3dev/wiki/Using-the-LEDs)
-* [Bluetooth](https://github.com/ev3dev/ev3dev/wiki/Using-Bluetooth)
-
-### Writing programs
-
-There are a number of programming languages available to use. The brick can
-run almost all languages that any other Linux distro can, so your favorite
-language is probably supported. Language bindings have already been written
-for many languages. You can learn more about the available libraries [here](/docs/libraries).
+The brick can run almost all languages that any other Linux distro can, so your
```

Comment:
might want to say "programming languages" in at least one place here so "languages" is not mistaken for i18n.